style.css

```diff
@@ -413,14 +413,182 @@ textarea::placeholder {
   opacity: 0.5;
 }
 
+.headerlist {
+  display: none;
+}
+
 @media (min-width: 375px) {
   .content-section {
     background-size: cover;
   }
 }
 
+/* desktop version */
+
 @media (min-width: 768px) {
+  .icon-Union {
+    display: none;
+  }
+
+  .topBar-nav {
+    position: sticky;
+    top: 0;
+    background-color: white;
+  }
+
+  .headerlist {
+    display: flex;
+    gap: 8px;
+    list-style-type: none;
+    padding-right: 20px;
+    font-family: var(--primary-font);
+    font-style: normal;
+    font-weight: 500;
+    font-size: 15px;
+    line-height: 20px;
+    color: #344563;
+    position: sticky;
+    top: 0;
+  }
+
+  .headerlist a:hover {
+    color: rgb(24, 155, 160);
+  }
+
   .content-section {
+    background-image: url(img/Header\ bg-desktop.png);
+    background-size: contain;
+    display: flex;
+    height: 718px;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    border-bottom-left-radius: 80px 80px;
+  }
+
+  .main-Container {
+    width: 60%;
+    margin-left: 0;
+    padding-top: 0;
+  }
+
+  .work-container {
+    background-color: #e5e5e5;
+    display: flex;
+    margin: 30px 10px;
+  }
+
```

Comment:
Kindly update your works section to match the figma design better🙏

Your design:
<img width="509" alt="image" src="https://user-images.githubusercontent.com/37116322/236272790-a7749f11-af72-4c01-9abc-9ffe6f79b96b.png">

Figma design:
<img width="493" alt="image" src="https://user-images.githubusercontent.com/37116322/236272912-780f308c-e9bb-4409-a293-2f127aa5a1e8.png">

A few things to note:
- Kindly increase the left and right margins applied to each card
- Please make sure that each card has the proper border-radius applied
- Please update the background color of each card
- Please make sure that the images, text and elements within the card are sized similar to the figma design 
- Please increase the spacing between the works section and about myself section